docs/_src/api/api/document_classifier.md

```diff
@@ -0,0 +1,110 @@
+<a name="base"></a>
+# Module base
+
+<a name="base.BaseDocumentClassifier"></a>
+## BaseDocumentClassifier Objects
+
+```python
+class BaseDocumentClassifier(BaseComponent)
+```
+
+<a name="base.BaseDocumentClassifier.timing"></a>
+#### timing
+
+```python
+ | timing(fn, attr_name)
+```
+
+Wrapper method used to time functions.
+
+<a name="transformers"></a>
+# Module transformers
+
+<a name="transformers.TransformersDocumentClassifier"></a>
+## TransformersDocumentClassifier Objects
+
+```python
+class TransformersDocumentClassifier(BaseDocumentClassifier)
+```
+
+Transformer based model for document classification using the HuggingFace's transformers framework
+(https://github.com/huggingface/transformers).
+While the underlying model can vary (BERT, Roberta, DistilBERT ...), the interface remains the same.
+This node classifies documents and adds the output from the classification step to the document's meta data.
+The meta field of the document is a dictionary with the following format:
+'meta': {'name': '450_Baelor.txt', 'classification': {'label': 'neutral', 'probability' = 0.9997646, ...} }
+
+With this document_classifier, you can directly get predictions via predict()
+
+Usage example:
+...
+retriever = ElasticsearchRetriever(document_store=document_store)
+document_classifier = TransformersDocumentClassifier(model_name_or_path="bhadresh-savani/distilbert-base-uncased-emotion")
+p = Pipeline()
+p.add_node(component=retriever, name="Retriever", inputs=["Query"])
+p.add_node(component=document_classifier, name="Classifier", inputs=["Retriever"])
+res = p.run(
+    query="Who is the father of Arya Stark?",
+    params={"Retriever": {"top_k": 10}, "Classifier": {"top_k": 5}}
+)
+print(res["documents"][0].to_dict()["meta"]["classification"]["label"])
+__Note that print_documents() does not output the content of the classification field in the meta data__
```

Comment:
Seems quite complicated here to print the meta data. Shall we just add an arg `print_meta=True/False` to `print_documents`? Could be helpful in other occasions as well and would simplify this example